tests/tests_e2e/test_e2e.py

```diff
@@ -15,13 +15,11 @@
 def test_app():
 
     # Load the environment variables.
-    file_path = find_dotenv('.env')
+    file_path = find_dotenv('.env', usecwd=True)
```

Comment:
I haven't seen this `usecwd` argument before - your tests pass for me with/without it so what's the use case here?

Reply:
It was necessary at one point during this module 9... not sure why it's not now!